### PR TITLE
updated mindsdb readme

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
## Description

updated mindsdb readme - https://github.com/mindsdb/mindsdb/blob/readme-update/README.md

Fixes #issue_number

- [ ] 📄 This change is a documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,14 @@
 
 <div align="center">
 
-
-
 <p>
-	<a href="https://github.com/mindsdb/mindsdb/actions"><img src="https://github.com/mindsdb/mindsdb/actions/workflows/release.yml/badge.svg" alt="MindsDB Release"></a>
+	<a href="https://github.com/mindsdb/mindsdb/releases"><alt="MindsDB Release"></a>
```
Comment (ZoranPandovski) [inline README.md:9]: This will not work since we need an image to be displayd

File: README.md [hunk 2]
```diff
@@ -28,124 +25,135 @@
...
+- Automated classification models
+- Automated regression models
+- Bring Your Own Model (BYOM) to MindsDB
 
+[Discover tutorials and use cases here](https://docs.mindsdb.com/use-cases/overview).
 
-[:question: :wave: Missing integration?](https://github.com/mindsdb/mindsdb/issues/new?assignees=&labels=&template=feature-mindsdb-request.yaml)
+## Get Started
 
+To get started, install MindsDB locally via [Docker](https://docs.mindsdb.com/setup/self-hosted/docker) or [Docker Extension](https://docs.mindsdb.com/setup/self-hosted/docker-desktop), following the instructions in linked doc pages.
```
Comment (ZoranPandovski) [inline README.md:99]: Maybe change Docker Extension to Docker Desktop

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (martyna-mindsdb) [reply README.md:99]: Done

File: README.md (hunk 1 above)
Comment (martyna-mindsdb) [reply README.md:9]: Do we have any image? Or shall I remove it?

Comment (ZoranPandovski) [reply README.md]: Release is the same as a PyPI version so we may change the PyPI btn to be releases and remove this one

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**